lighthouse-core/scripts/legacy-javascript/summary-signals.json

```diff
@@ -137,45 +29,33 @@
       "name": "core-js-2-only-polyfill/es6-array-from",
       "signals": "Array.from"
     },
-    {
-      "name": "core-js-2-only-polyfill/es6-array-index-of",
-      "signals": ""
```

Comment:
this were empty because legacy javascript never checked for these polyfills, because I omitted them intentionally. The duplication of the polys in the `run.js` script had some extra stuff. fixed now they it uses the same data source.